Bump(pyavd): Make pyavd-utils a proper dependency (#6171)
aristanetworks/ansible-avd: docs/release-notes/6.x.x.md

### Changes to requirements

- Python requirements:

>   - PyAVD now requires the `pyavd-utils` library. This library is maintained alongside `pyavd` and co-versioned. It provides Python binding to Rust code. This should not have any impact

Reviewer: ```suggestion
  - PyAVD now requires the `pyavd-utils` library. This library is maintained alongside `pyavd` and is co-versioned. It provides Python bindings to Rust code. This should not have any impact
```

>     if you are installing `pyavd` from pip, everything will install transparently. If you are a developer contributing to AVD, you will need to install `cargo`. Refer to the contribution guide.

Reviewer: ```suggestion
    if you are installing `pyavd` with `pip`; everything will install transparently. If you are a developer contributing to AVD, you will need to install `cargo`. Refer to the [contribution guide](../contribution/overview.md).
```
